### PR TITLE
Add new student fix

### PR DESCRIPTION
**Pull request by armandodlvr**

### CODE REVIEW

**Comment by imcodingideas** [comment]
This fixes: https://github.com/freeCodeCamp/classroom-mode/issues/65 @user512

File: .gitignore [hunk 1]
```diff
@@ -24,4 +24,6 @@ client/.env.production.local
 client/npm-debug.log*
 npm-debug.log*
 package-lock.json
+server/package-lock.json*
+client/package-lock.json*
```
Comment (user512) [inline .gitignore]: Should we not commit these file that specify dependency?
According to npm, looks like they recommend committing this file.
https://github.com/npm/npm/blob/v5.8.0/doc/files/package-lock.json.md

**Review by user512 — COMMENTED** [review]
@armandodlvr, I pulled down the branch and found out I can no longer add any student, do you mind verifying this?

File: .gitignore (hunk 1 above)
Comment (armandodlvr) [reply .gitignore]: Fixed.

**Comment by armandodlvr** [comment]
Fixed

File: .gitignore (hunk 1 above)
Comment (user512) [reply .gitignore]: Do you think we should not put `package-lock.json*` in `.gitignore` as NPM suggested?

File: server/controllers/studentController.js [hunk 2]
```diff
@@ -76,24 +76,25 @@ exports.addStudent = (req, res) => {
...
+            Student.create({
+              name: name,
+              username: username,
+              email: email,
+              completedChallengesCount: fccResults.completedChallenges && fccResults.completedChallenges.length,
+              completedChallenges: fccResults.completedChallenges
+            }).catch((error) => {
+              res.status(422).json({ error })
+            })
+            console.log("entro");
```
Comment (user512) [inline server/controllers/studentController.js]: Is this redundant? Can we remove this?

**Review by user512 — COMMENTED** [review]
Can't test due to FCC outage, but code looks good, do you mind doing some code clean up as suggested?

**Comment by armandodlvr** [comment]
Done: #65 @user512

**Pull request closed without merging**